### PR TITLE
LPS-198024 [Marketplace] Adjustments to the Solution app card

### PR DESCRIPTION
**Pull request by juliocezar96**
subtask : [LPS-198024](https://liferay.atlassian.net/browse/LPS-198024)

### CODE REVIEW

**Comment by juliocezar96** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-198024 [Impedibug] Adjustments to the Solution app card</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-198024">[Impedibug] Adjustments to the Solution app card</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Impedibug</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Low</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>Open</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Support QA</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Luana Silva</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

1 file changed, 24 insertions(+), 19 deletions(-)

**Comment by juliocezar96** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 1]
```diff
@@ -108,15 +112,15 @@
 									<#assign productPriceModels = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
 
 									<#list productPriceModels as productPriceModel>
-										<div class="color-neutral-3 font-size-paragraph-small mt-1">
+										<div class="color-neutral-3 font-size-paragraph-small mt-1 developer-name">
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:115]: This is weird, you are assigning the color #545d69 but you have `color-neutral-3` applied to this div, which color should be used?

```suggestion
										<div class="color-neutral-3 font-size-paragraph-small mt-1 developer-name">
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 2]
```diff
@@ -49,6 +49,10 @@
 		color: #545d69;
 	}
 
+	.developer-name{
+		color: #545d69;
+	}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:54]:
```suggestion
	.developer-name {
		color: #545d69;
	}
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 3]
```diff
@@ -90,11 +94,11 @@
 						productSpecifications=product.productSpecifications![]
 						productURL=portalURL?replace("solutions-marketplace", "p" ) + "/" + product.urls.en_US />
 
-					<a class="solution-search-results-card bg-white border-radius-medium d-flex flex-column mb-0 p-3 text-dark text-decoration-none" href=${productURL}>
-						<div class="align-items-center d-flex image-container justify-content-center rounded">
+					<a class="solution-search-results-card bg-white border-radius-medium d-flex flex-column mb-0 rounded text-dark text-decoration-none " href=${productURL}>
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:97]: Remove blank space

```suggestion
					<a class="solution-search-results-card bg-white border-radius-medium d-flex flex-column mb-0 rounded text-dark text-decoration-none" href=${productURL}>
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by juliocezar96** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:test:relevant

**Review by ytuik — APPROVED** [review]
LGTM

**Comment by LuanaKSilva** [comment]
ci:test:relevant

**Comment by juliocezar96** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**